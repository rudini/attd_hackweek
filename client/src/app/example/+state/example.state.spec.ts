import { exampleReducer, ExampleAction } from './example.state';

describe('example reducer test', () => {
    it('should correctly add to collection', () => {
        const added = exampleReducer({ collection: [1] }, ExampleAction.addToCollection(2));
        expect(added).toMatchSnapshot();
    });
    it('should correctly remove from collection', () => {
        const added = exampleReducer({ collection: [1, 2, 3] }, ExampleAction.removeFromCollection(1));
        expect(added).toMatchSnapshot();
    });
});
