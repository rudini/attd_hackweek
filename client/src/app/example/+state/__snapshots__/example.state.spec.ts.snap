// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`example reducer test should correctly add to collection 1`] = `
Object {
  "collection": Array [
    1,
    2,
  ],
}
`;

exports[`example reducer test should correctly remove from collection 1`] = `
Object {
  "collection": Array [
    1,
    3,
  ],
}
`;
